gilded-rose-llc-frontend/components/item-block.tsx

```diff
@@ -5,11 +5,13 @@ interface ItemBlockProps {
 
 const ItemBlock = ({ name, price }: ItemBlockProps) => {
   return (
-    <li className="bg-[#8690F4] my-2">
-      <h2>{name}</h2>
-      <h3>{price}</h3>
-      <button>add to cart</button>
-    </li>
+    <div className="rounded-[5px] box-border px-10 py-6 h-full space-y-5 bg-[#8690F4] flex flex-col items-center justify-between">
```

Comment:
Question: Are these tailwind specific styles?

Reply:
Yes, tailwind specific

Reply:
Something to think about: It gets pretty messy with all these inline styling, some of them are repeated too. I wonder if the className can be something like: `className="inventory__item-name"`([BEM convention](https://en.bem.info/methodology/naming-convention/)) and then have the inline styles moved to a separate css file.